### PR TITLE
wasm(32|64): update alignment string

### PR DESCRIPTION
**Pull request by durin42**
See llvm/llvm-project@c5ab70c508457eaece5d7ff4ab79a2f90bc67f06

@rustbot label: +llvm-main

### CODE REVIEW

**Comment by jieyouxu** [comment]
LLVM PR for backlink: https://github.com/llvm/llvm-project/pull/119204

File: compiler/rustc_codegen_llvm/src/context.rs [hunk 1]
```diff
@@ -164,6 +164,11 @@ pub(crate) unsafe fn create_module<'ll>(
             // See https://github.com/llvm/llvm-project/pull/118004
             target_data_layout = target_data_layout.replace("-i128:128", "");
         }
+        if sess.target.arch.starts_with("wasm32") {
+            // LLVM 20 updates the powerpc64 layout to correctly align 128 bit integers to 128 bit.
+            // See https://github.com/llvm/llvm-project/pull/118004
```
Comment (jieyouxu) [inline compiler/rustc_codegen_llvm/src/context.rs:169]: Question: copy-pasta'd comment? This should be https://github.com/llvm/llvm-project/pull/119204 right?

**Review by jieyouxu — APPROVED** [review]
Thanks, one comment issue then r=me after PR CI is green.

File: compiler/rustc_codegen_llvm/src/context.rs (hunk 1 above)
Comment (durin42) [reply compiler/rustc_codegen_llvm/src/context.rs]: sigh, yep - thanks!

(our LLVM HEAD CI spotted another case too, so I fixed that as well)

**Review by jieyouxu — APPROVED** [review]
Thanks, LGTM

**Comment by jieyouxu** [comment]
@durin42 can you also update the PR title and reflection, since we also now modify wasm64 targets?

**Comment by durin42** [comment]
> @durin42 can you also update the PR title and reflection, since we also now modify wasm64 targets?

I'm not sure what you mean by reflection, but I updated the PR title

**Comment by jieyouxu** [comment]
> I'm not sure what you mean by reflection, but I updated the PR title

I managed to type "reflection" when I meant "description", lol

**Comment by durin42** [comment]
force-pushed an updated log message

**Pull request merged**